tensorrt_llm/_torch/model_config.py

```diff
@@ -176,6 +176,7 @@ def is_generation_model(model_architectures: Optional[List[str]]) -> bool:
     def from_pretrained(cls,
                         checkpoint_dir: str,
                         trust_remote_code=False,
+                        force_dynamic_quantization=False,
```

Comment:
I am not a huge fan of exposing this parameter explicitly to the `from_pretrained` function. The `from_pretrained` function has a distinct notion in the HuggingFace/Transformers library and we are obviously emulating it here. It doesn't